tools/topology/sof-tgl-rt711-rt1308.m4

```diff
@@ -0,0 +1,245 @@
+	2, 1, 2, s32le,
+	1000, 0, 0, ALH, 3, s32le, 3,
+	48000, 48000, 48000)
+
+# Low Latency playback pipeline 3 on PCM 2 using max 2 channels of s24le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_DAI_ADD(sof/pipe-volume-playback.m4,
+	3, 2, 2, s24le,
+	1000, 0, 0, SSP, 2, s24le, 3,
+	48000, 48000, 48000)
+
+# Passthrough capture pipeline 4 on PCM 3 using max 4 channels.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_DAI_ADD(sof/pipe-passthrough-capture.m4,
+	4, 3, 4, s32le,
+	1000, 0, 0, DMIC, 0, s32le, 3,
+	48000, 48000, 48000)
+
+# Passthrough capture pipeline 5 on PCM 4 using max 4 channels.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_DAI_ADD(sof/pipe-passthrough-capture.m4,
+	5, 4, 4, s16le,
+	1000, 0, 0, DMIC, 1, s16le, 3,
+	16000, 16000, 16000)
+
+ifelse(HDMI, `1',
+`
+# Low Latency playback pipeline 6 on PCM 5 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	6, 5, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Low Latency playback pipeline 7 on PCM 6 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	7, 6, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Low Latency playback pipeline 8 on PCM 7 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	8, 7, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Low Latency playback pipeline 9 on PCM 8 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	9, 8, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+')
+
+#
+# DAIs configuration
+#
+
+dnl DAI_ADD(pipeline,
+dnl     pipe id, dai type, dai_index, dai_be,
+dnl     buffer, periods, format,
+dnl     deadline, priority, core, time_domain)
+
+# playback DAI is ALH(SDW0 PIN2) using 3 periods
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	1, ALH, 2, SDW0-Playback,
+	PIPELINE_SOURCE_1, 3, s32le,
+	1000, 0, 0)
+
+# capture DAI is ALH(SDW0 PIN2) using 3 periods
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
+	2, ALH, 3, SDW0-Capture,
+	PIPELINE_SINK_2, 3, s32le,
+	1000, 0, 0)
+
+# playback DAI is SSP2 using 3 periods
+# Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	3, SSP, 2, SSP2-Codec,
+	PIPELINE_SOURCE_3, 3, s24le,
+	1000, 0, 0)
+
+# capture DAI is DMIC01 using 3 periods
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
+	4, DMIC, 0, dmic01,
+	PIPELINE_SINK_4, 3, s32le,
+	1000, 0, 0)
```

Comment:
Please update to use `SCHEDULE_TIME_DOMAIN_TIMER`.
Master branch has been updated in the meantime.